core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -129,7 +129,7 @@ THE SOFTWARE.
       <j:forEach var="c" items="${computers}">
```

Comment:
When navigating to `jenkins/computer/Foo/`, `${computers}` does not include the master node, so it will not be displayed on that page regardless of its online status. Is it fine to leave that behavior, or should we add the master node to `computers` when it is offline?